### DISCUSSION TITLE
Amber Electric Dashboard for Tablets/Browsers

### DISCUSSION THREAD

**Discussion by GrahamLea** [amberelectric/public-api#111] [Show and tell]
I've been wanting an Amber dashboard that I can run on an old tablet, stick to the kitchen wall, and see the current price and future forecasts from across the room.

I've built it as a a responsive webpage that should work on any device with a modern browser, and chucked it online here:

   https://grahamlea.github.io/amber-dashboard/

You just have to plop in your API Key (stored locally + securely in your browser) and it should go!

<img width="1488" alt="image" src="https://user-images.githubusercontent.com/754403/167063406-28e148d1-d581-4aa1-96ed-08a98890a574.png">

**Comment by madpilot** [comment]
Awesome! Is the code available for those that want to check out how you did it?

**Comment by GrahamLea** [comment]
Yep, code is here: https://github.com/GrahamLea/amber-dashboard/
